docs/ops/quantization/FakeQuantize_1.md

```diff
@@ -37,29 +37,34 @@ else:
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>
-    * *pdpd* - PaddlePaddle-style implicit broadcasting.
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
+    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
```

Comment:
I've got [a comment](https://github.com/openvinotoolkit/openvino/pull/6512#discussion_r667890708) that we can remove  "**Default value**: None" from the "**Required**: *yes*" attributes